pwa/pages/reparateur/inscription/demande-enregistree.tsx

```diff
@@ -0,0 +1,122 @@
+import {NextPageWithLayout} from '@interfaces/NextPageWithLayout';
+import React, {useContext, useEffect} from 'react';
+import {RepairerRegistrationContext} from '@contexts/RepairerRegistrationContext';
+import {useRouter} from 'next/router';
+import {Box, Paper, Button, Container} from '@mui/material';
+import Link from 'next/link';
+import RepairerRegistrationLayout from '@components/layout/RepairerRegistrationLayout';
+
+const RepairerSuccessRegistration: NextPageWithLayout = () => {
+  const {
+    stepOneCompleted,
+    stepTwoFirstQuestionCompleted,
+    stepTwoCompleted,
+    formCompleted,
+    success,
+    setStepOneCompleted,
+    setStepTwoFirstQuestionCompleted,
+    setStepTwoCompleted,
+    setFormCompleted,
+  } = useContext(RepairerRegistrationContext);
+  const router = useRouter();
+
+  useEffect(() => {
+    if (!formCompleted) {
```

Comment:
You can either only checks for `if (!formCompleted)` or ensure that all steps are completed by checking
```
if (!formCompleted || !stepOneCompleted || ! stepTwo...)
```